server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -99,6 +122,7 @@ export function generateToken(
 	lifetime: number = 60 * 60,
 	ver: string = "1.0",
 	jti: string = uuid(),
+	fluidRelayKeylessAccess = false,
```

Comment:
Should we update the same function in services-client?

Reply:
services-client need not use this claim. This would be consumed only by the server. Hence, I did not update the services-client package.